modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/ColorSwatchItem.ts

```diff
@@ -53,3 +53,15 @@ export const renderColorSwatchItem = (spec: Menu.FancyMenuItem, backstage: UiFac
     ]
   };
 };
+
+const getcolorItems = (spec: Menu.FancyMenuItem, backstage: UiFactoryBackstage): Menu.ChoiceMenuItemSpec[] => {
+  if (Type.isNonNullable(spec.initData)) {
+    if (spec.initData.colorselection) {
```

Comment:
No falsy checks please! Use something explicit like`Type.isObject` instead to make sure it's what we want.

Additionally, there's a typo here as this should be `colourSelection`. In saying that though, `selection` is confusing and not needed I believe. Instead, might I recommend just calling this `colors`?